package/kernel/linux/modules/tpm.mk

```diff
@@ -0,0 +1,36 @@
+TPM_MENU:=TPM
+
+define KernelPackage/tpm
```

Comment:
OK, shall do. Note I have added a TPM menu as I am trying to follow roughly the same sections as the kernel uses and the SSL menu certainly didn't seem like the right place. Any suggestions or are we OK with this?